app/components/wallet/key-import/WalletKeyImportDialog.js

```diff
@@ -42,8 +42,8 @@ const messages = defineMessages({
   },
   passwordSwitchPlaceholder: {
     id: 'wallet.key.import.dialog.passwordSwitchPlaceholder',
-    defaultMessage: '!!!Activate to create password',
-    description: 'Text for the "Activate to create password" switch in the wallet key import dialog.',
+    defaultMessage: '!!!Activate to create password. Note that password needs to be at least 7 characters long, and have at least 1 uppercase, 1 lovercase letter and 1 number.',
+    description: 'Text for the "Activate to create password. Note that password needs to be at least 7 characters long, and have at least 1 uppercase, 1 lovercase letter and 1 number." switch in the wallet key import dialog.',
```

Comment:
@nikolaglumac Please reuse the translation from the change password dialog everywhere else (move it to more generic place) and add a dedicated paragraph … it doesn't look very nice together with the "Activate to create password." etc.